src/lib.rs

```diff
@@ -8,6 +8,7 @@
 #![feature(external_doc)]
 #![doc(include = "../README.md")]
 #![warn(missing_docs)]
+#![feature(ordering_helpers)]
```

Comment:
This is not needed anymore in the latest nightly. If it's kept just for the formatter is fine, otherways, it can be removed since is stable since `1.53`

Reply:
I just updated the nightly and the feature is still required